src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Main {
+public static void main(String[] args) {
```

Comment:
The `main` method is empty. You should implement this method to demonstrate the functionality of your application, such as creating instances of figures and using their methods.